go.mod

```diff
@@ -169,3 +169,6 @@ replace github.com/gin-gonic/gin => github.com/gin-gonic/gin v1.7.0
 
 // TODO: Required for v1.13.x: Remove this and update the require line above to have the new version once it's out. https://github.com/provenance-io/provenance/issues/1015
 replace github.com/CosmWasm/wasmd => github.com/provenance-io/wasmd v0.28.0-pio-1-rc1
+
+// TODO: Required for IBC Transfer Module. Remove this when they patch it to include our changes. https://github.com/provenance-io/provenance/issues/1100
+replace github.com/cosmos/ibc-go/v5 => github.com/provenance-io/ibc-go/v5 v5.0.0-pio
```

Comment:
Both the tag and branch name are the same: `v5.0.0-pio`.  That's causing the proto-update-deps script to die because this command:
```
> curl -sSL github.com/provenance-io/ibc-go/tarball/v5.0.0-pio
```
returns this:
```
the given path has multiple possibilities: #<Git::Ref:0x00007fb45395f8a8>, #<Git::Ref:0x00007fb453967a58>
```
 
Retagging it as `v5.0.0-pio-1` and switching to that version should get around that.